checkbox.js

```diff
@@ -0,0 +1,125 @@
+/*jshint esversion: 6*/
+/*global document*/
+/*global alert*/
+
+class Data {
+    static getCheckboxList() {
+        return ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday', 'None'];
+      } 
+  }
+  
+  class CheckboxOperations {
+    
+    createCheckboxList() {
+      let ul = this.createUlElementForCheckbox();
+      document.body.appendChild(ul);
+        this.createAndAppendCheckboxListTo(ul);
+    }
+  
+    createAndAppendCheckboxListTo(ul) {
+        let checkboxIds = Data.getCheckboxList();
+        checkboxIds.forEach((checkboxId) =>{
+        let label = this.createLabelElement(checkboxId);
+              let checkbox = this.createCheckbox(checkboxId);
+        
+        label.appendChild(checkbox);
+        label.appendChild(document.createTextNode(checkboxId));
+        
+        ul.appendChild(this.createLiElementAndAppend(label));
+  
+      });
+      }
+    createUlElementForCheckbox() {
+        let ul = document.createElement('ul');
+      ul.style.listStyle = 'none';
+        return ul;
+      }
+  
+      createLiElementAndAppend(label) {
+      let li = document.createElement('li');
+      li.appendChild(label);
+      return li;
+    }
+  
+      createCheckbox(checkboxId) {
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.name = checkboxId;
+      checkbox.value = checkboxId;
+      checkbox.id = checkboxId;
+      checkbox.style.margin = '10px';
+      return checkbox;
+    }
+  
+      createLabelElement(labelId) {
+      const label = document.createElement('label');
+      label.setAttribute('for', labelId);
+      return label;
+    }
+  
+    makeAllCheckboxUnchecked() {
+    let checkboxes = document.querySelectorAll('input');
+    checkboxes.forEach((checkbox) => {
+      checkbox.checked = checkbox.id === "None";
+    });
+  }
+  
+   isMoreThanThreeCheckboxChecked(id) {
```

Comment:
Rather than running this on every time. Cant we simply maintain a class variable containing the selected days.